chromium_src/chrome/DEPS

```diff
@@ -1,2 +1,7 @@
 from brave_chromium_utils import inline_file
 inline_file("//chrome/DEPS", globals(), locals())
+
+include_rules = [
+  # We need to be able to import from chrome to write overrides.
+  "+chrome",
```

Comment:
this is not correct and would allow things like chrome/renderer inside chrome/browser. What is the actual include you are having trouble with?